src/main/java/core/basesyntax/model/FruitTransactionDto.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.model;
+
+import com.opencsv.bean.CsvBindByPosition;
+
+public class FruitTransactionDto {
+    @CsvBindByPosition(position = 0)
```

Comment:
I don't understand what this annotation is for. 
Is this your custom annotation or do you use an additional library to parse CSV?

Reply:
Its OpenCSV library